about.html

```diff
@@ -39,31 +44,18 @@
                     <a href="#" class="nav-link" id="cc-campaign">CC&nbsp;&nbsp;Campaign</a>
                 </li>
             </ul>
-            <div class="hamburger">
-                <span class="bar"></span>
-                <span class="bar"></span>
-                <span class="bar"></span>
-            </div>
         </nav>
     </section>
 
     <section class="home">
-        <p class="hello-title"></p>
+        <p class="hello-title">"Hello! Sharing world"</p>
         <h1 class="head-title">CREATIVE COMMONS<br>GLOBAL SUMMIT 2023</h1>
```

Comment:
- [ ] Great instinct using a `hx` tag but please, remember that it good to have just one `h1` title in our entire document which we already have in the homepage. Kindly opt for any other from the range of `h2` - `h6` tags

Reply:
Thanks